packages/aragon-wrapper/src/index.js

```diff
@@ -859,6 +859,32 @@ export default class Aragon {
     })
   }
 
+  /**
+   * Initialize the appMetadata observable
+   */
+  initAppMetadata () {
+    this.appMetadata = new BehaviorSubject({}).pipe(
+      publishReplay(1)
+    )
+    this.appMetadata.connect()
+  }
+
+  /**
+   * registers new app metadata item
+   *
+   * @param {string} from
+   * @param {string} dataId
+   * @param {string} cid
+   */
+  registerAppMetadata (from, dataId, cid) {
+    this.registerMetadata.next({
+      from,
+      to: ['*'],
```

Comment:
This looks like it's hard-coded, and can't be change by passing in a custom "to" param

Reply:
Changed from hardcoded to default parameters in both wrapper and api.